### ISSUE TITLE
movie extras button

### ISSUE THREAD

**Issue by Aemstel2** [jurialmunkey/skin.arctic.fuse#540]
### Skin section

Video Info Dialog

### Feature description

Have a button on the left sidemenu on the movie information screen, that shows the videos that are in the extras folder of a movie (usually dvd/blu-ray bonus material). Maybe it could function/look the same as the trailers button.

### Why is this feature needed? Please justify and explain.

I think it would be a nice addition for people using this feature. Right now I think the only way to access these extras is through the context menu.

### Checklist

- [X] I have made a separate issue for each feature I am requesting.
- [X] I have searched the issues and this feature has not been requested.
- [X] I am using the most recent skin version.

**Comment by jurialmunkey** [comment]
Yeah I've been meaning to add this in. Just waiting for proper Omega release version since it won't work for Nexus and I want to keep things working relatively okay for Nexus while it is still the main version. Once Omega is properly out I will add this in.

**Comment by Aemstel2** [comment]
Great, thanks!
